app/controllers/application_controller/ci_processing.rb

```diff
@@ -22,6 +22,8 @@ def set_ownership
                  else
                    request.parameters[:controller]
                  end
+    @edit ||= {}
+    @edit[:controller] = controller
```

Comment:
I wouldn't bother here, let's just stick to the version that's known to work :).